prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,38 @@
+let number1 = 0;
+let number2 = 1;
+const product = number1 * number2;
+console.log('Product: ' + product);
+console.log('typeof for product is: ' + typeof product);
+
+let charge = 5;
+let payment = 1;
+let amountRemaining = payment - charge;
+console.log('Amount remaining: ' + amountRemaining);
+console.log('typeof for amountRemaining is: ' + typeof amountRemaining);
+
+let tests = 5;
+let assignments = 10;
+let final = 1;
+let grade = (tests + assignments + final) / 3;
+console.log('Grade: ' + grade);
+console.log('typeof for grade is: ' + typeof grade);
+
+let firstName = 'Kyle';
+let lastName = 'Spence';
+let fullName = firstName + ' ' + lastName;
+console.log('My name is ' + fullName);
+console.log('typeof for fullName is: ' + typeof fullName);
+
+let pH = 15;
+let isAcidic = pH < 7;
+console.log('Is it acidic: ' + isAcidic);
+console.log('typeof for isAcidic is ' + typeof isAcidic);
+
+let numberOfSoldiers = 1000;
+let isSparta = numberOfSoldiers === 300;
+console.log('Are we cooler than Sparta at Thermopylae: ' + isSparta);
+console.log('typeof for isSparta is: ' + typeof isSparta);
+
+let nickname = fullName + ' is the GOAT!';
```

Comment:
This should be broken up into 2 steps. You should assign `fullName` to `nickname` first. And then on another line use `+=` to add the string